src/rulesets/rest/2022-05-25/property-rules.ts

```diff
@@ -133,15 +133,31 @@ const enumOrExample = new RequestRule({
   },
 });
 
-const dateFormatting = new RequestRule({
+const requestDateFormatting = new RequestRule({
   name: "request property date formatting",
   docsLink: links.standards.formats,
   rule: (requestAssertions) => {
     requestAssertions.property.added("use date-time for dates", (property) => {
-      if (["created", "updated", "deleted"].includes(property.value.key)) {
+      if (property.value.key.endsWith("_at")) {
         if (property.value.flatSchema.format !== "date-time") {
           throw new RuleError({
-            message: "expected property to have formate date-time",
+            message: "expected property to have format date-time",
+          });
+        }
+      }
+    });
+  },
+});
+
+const responseDateFormatting = new ResponseBodyRule({
+  name: "response property date formatting",
+  docsLink: links.standards.formats,
+  rule: (responseAssertions) => {
+    responseAssertions.property.added("use date-time for dates", (property) => {
+      if (property.value.key.endsWith("_at")) {
+        if (property.value.flatSchema.format !== "date-time") {
+          throw new RuleError({
+            message: "expected property to have format date-time",
```

Comment:
Assuming this is the error message a user sees, it might be more user-friendly to be specific, e.g. `properties ending in _at expected to have format date-time`

Reply:
Good point... messages should be transparent so it doesn't become a game of "appease the mysterious oracle". I could also link to the documentation on this.